lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -444,6 +446,9 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
         assert self._rm_client
         assert self._azure_runbook
 
+        platform_message = PlatformMessage(status=PlatformStatus.DEPLOYING, name=AZURE)
```

Comment:
Those messages can be put in base platform, so every platform can get it.

Reply:
If we put these initialization in the base platform, we would not have access to the name of platform.

Reply:
Try self.type_name(). For initialization, it may not be able to in `_initialize`, try put it in `prepare_environment` after `self.initailize()`.